Kiss.Bff.EndToEndTest/Beheer/GesprekresultatenScenarios.cs

```diff
@@ -0,0 +1,169 @@
+        }
+
+        [TestMethod("3. Editing an existing Gesprekresultaten")]
+        [DataRow("Automation Gesprekresultaten edit")]
+        public async Task EditKanaal(string originalGesprekresultaten)
+        {
+            // Precondition: Add the Gesprekresultaten
+            await AddGesprekresultatenHelper(originalGesprekresultaten);
+
+            string updatedGesprekresultaten = "Automation Gesprekresultaten Updated";
+
+
+            try
+            {
+                await Step("Given the user is on the 'Gesprekresultaten' section of the 'Beheer' tab");
+
+                await Page.GotoAsync("/");
+                await Page.GetByRole(AriaRole.Link, new() { Name = "Beheer" }).ClickAsync();
+                await Page.GetByRole(AriaRole.Link, new() { Name = "Gespreksresultaten" }).ClickAsync();
+
+                await Step($"When user clicks on channel list with name as '{originalGesprekresultaten}'");
+
+                await Page.GetByRole(AriaRole.Link, new() { Name = originalGesprekresultaten }).ClickAsync();
+
+                await Step($"And user updates title to '{updatedGesprekresultaten}'");
+
+                await Page.GetByRole(AriaRole.Textbox, new() { Name = "Titel" }).FillAsync(updatedGesprekresultaten);
+
+                await Step("And user clicks on Opslaan");
+
+                await Page.GetOpslaanButton().ClickAsync();
+
+                await Step($"And updated channel '{updatedGesprekresultaten}' is added to the list of Kanalen");
+
+                await Expect(Page.GetByRole(AriaRole.Listitem).Filter(new() { HasText = updatedGesprekresultaten })).ToBeVisibleAsync();
+            }
+            finally
+            {
+                await DeleteGesprekresultatenHelper(updatedGesprekresultaten);
+            }
+        }
+
+        [TestMethod("4. Deleting a Gespreksresultaten")]
+        [DataRow("Automation Gesprekresultaten delete")]
+        public async Task DeleteKanaal(string deleteGesprekresultaten)
+        {
+            await Step("Precondition: Automation Gesprekresultaten is created");
+            await AddGesprekresultatenHelper(deleteGesprekresultaten);
+
+            try
+            {
+                await Step("When the user deletes the Gesprekresultaten");
+                await DeleteGesprekresultatenHelper(deleteGesprekresultaten);
+            }
+            finally
+            {
+
+                var GespreksresultatenExists = await Page.GetByRole(AriaRole.Listitem)
+                    .Filter(new() { HasText = deleteGesprekresultaten })
+                    .IsVisibleAsync();
+
+                if (GespreksresultatenExists)
+                {
+                    await DeleteGesprekresultatenHelper(deleteGesprekresultaten);
+                }
+            }
+        }
+
+
+        // Helper method to add a new Gesprekresultaten
+        private async Task AddGesprekresultatenHelper(string Gesprekresultaten)
```

Comment:
gespreksresultaten is plural. this method is are adding one. It's better to call in AddGesprekresultaatHelper